tests/data/Si_phonons/si.dyn0

```diff
@@ -0,0 +1,5 @@
+   2   2   2
```

Comment:
Please add QE input files as well as pseudopotential file for the phonon test data